camel/generators.py

```diff
@@ -150,19 +152,19 @@ def __init__(self, assistant_role_names_path:
 
         if assistant_role_names is None:
             with open(assistant_role_names_path, "r") as f:
-                assistant_role_names: List[str] = f.read().splitlines()
+                assistant_role_names_: List[str] = f.read().splitlines()
```

Comment:
Why do we need this underscore?

Reply:
Otherwise mypy complains that assistant_role_names shadows another assistant_role_names from earlier on in the function.

Reply:
I could not come up with a good suffix :)